templates/webTemplateFiles.yaml

```diff
@@ -126,42 +126,42 @@
   description: Provides templates for miscellaneous use cases
 - lang: go
   id: xylo.numbers
-  url: https://raw.githubusercontent.com/xylo/intellij-postfix-templates/master/templates/go/numbers.postfixTemplates
+  url: https://raw.githubusercontent.com/philipgriggs/go-postfix-templates/master/numbers.postfixTemplates
   author: Philip Griggs
   email: philipgriggs@gmail.com
   website: https://github.com/xylo/intellij-postfix-templates/wiki/Go-web-templates:-xylo.numbers
```

Comment:
The wiki entries aren't there, yet. Easiest just link to your new repo and document the usage in the Readme.

Reply:
Yes. The wiki links should also point to the new repo.